pkg/action/list.go

```diff
@@ -242,6 +245,20 @@ func filterList(releases []*release.Release) []*release.Release {
 	return list
 }
 
+func (l *List) filterStateMask(releases []*release.Release) []*release.Release {
+	desiredStateReleases := make([]*release.Release, 0)
+
+	for _, rls := range releases {
+		currentStatus := l.StateMask.FromName(rls.Info.Status.String())
+		if l.StateMask&currentStatus == 0 {
```

Comment:
Can we put a space between statemask, "&", and currentStatus for readability?

Reply:
Hi @jdolitsky -- thanks for the review.

It looks like gofmt wants to remove the spaces when the `&` operator is used in this if statement ([here's](https://github.com/helm/helm/blob/70f89e5f26015a92432bdeb94ddb6d6e52532b19/pkg/action/list_test.go#L51) a similar formatting elsewhere in the codebase, presumably from gofmt). I'm reluctant to fight gofmt here, but I'll defer to your judgement.

It looks like gofmt will keep the spaces around `&` when used in an assignment, however.
```
// https://play.golang.org/p/XmQYHSKQT5B
package main

import (
	"fmt"
)

func main() {
	a := 100
	b := 1

	if a&b == 0 {
		fmt.Println("foo")
	}

	fmt.Printf("%b\n", a&b)

	out := a & b
	fmt.Printf("%b\n", out)
}
```


We could break the condition out into a variable assignment on the line above to improve readability, would you prefer that?
```suggestion
		shouldInclude := l.StateMask & currentStatus
		if shouldInclude {
```

Reply:
my proposed change above isn't quite right -- should have taken my time.

gofmt removes the spaces around `&` if the assignment includes the equality operator
```
shouldNotInclude := l.StateMask&currentStatus == 0
```
in order to maintain the space around `&`, it would need to be something like
```

mask := l.StateMask & currentStatus
if mask == 0 {
...
```

Reply:
I agree, lets not fight with gofmt :)

I personally think the `mask := l.StateMask & currentStatus` is more readable. The change overall look fine to me

Reply:
addressed in f9d3524 @jdolitsky 👍